fiasko_bro/code_validator.py

```diff
@@ -9,6 +10,24 @@ def validate_repo(path_to_repo, path_to_original_repo=None):
     return code_validator.validate(path_to_repo, path_to_original_repo)
 
 
+def py_files_count(path_to_repo):
```

Comment:
Also, this function is generic enough to work for any directory, not only `path_to_repo`. So let's rename the argument to `directory` or something like that.